rrfs-test/IODA/yaml/bufr_ncep_nexrad.yaml

```diff
@@ -0,0 +1,160 @@
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./testinput_20230610/rtma_ru.t0000z.nexrad.tm00.bufr_d"
+
+      exports:
+        group_by_variable: spectralWidth 
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+          longitude:
+            query: "[*/CLON, */CLONH]"
+          latitude:
+            query: "[*/CLAT, */CLATH]"
+          stationIdentification:
+            query: "*/SSTN"
+
+          # ObsValue
+          height: 
+            query: "*/HSMSL"
+          heightOfAntenna: 
+            query: "*/HSALG"
+          beamAzimuthAngle: 
+            query: "*/ANAZ"
+          beamTiltAngle: 
+            query: "*/ANEL"
+          gateRange: 
+            query: "*/NL2RW/DIST125M"
+            transforms:
+              - scale: 1000
+          radialVelocity: 
+            query: "*/NL2RW/DMVR"
+          spectralWidth: 
+            query: "*/NL2RW/DVSW"
+          ppiVolume: 
+            query: "*/VOID"
+          ppiIndex: 
+            query: "*/SCID"
```

Comment:
Just curious what these variables are and if we need them? They seem to be an index number of some kind.